src/main/java/io/codelex/collections/practice/Flight/Program.java

```diff
@@ -0,0 +1,131 @@
+package io.codelex.collections.practice.Flight;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+
+public class Program {
+    private static final String file = "C:\\Users\\ReinisLubins\\Desktop\\PROGRAMESANA\\JAVA SYLABUS\\codelex-exercises\\src\\main\\resources\\collections\\flights.txt";
+    private static final List<String> flights = new ArrayList<>();
+    private static final Map<Integer, String> allCities = new HashMap<>();
+    private static final Map<Integer, String> listOfAvailableFlights = new HashMap<>();
+    private static final List<String> chosenRoute = new ArrayList<>();
+
+    private static void fileRead() throws IOException {
+        BufferedReader bf = new BufferedReader(new FileReader(file));
+        String lines = bf.readLine();
+
+        while (lines != null) {
+            flights.add(lines);
+            lines = bf.readLine();
+        }
+    }
+
+    private static String flightFrom(int index) {
+        String[] ir = flights.get(index).replaceAll(" ", "").split("->");
```

Comment:
`ir` ? 😄

Reply:
Pa to biju aizmirsis, kad visu veidoju XD. Kļūdas izlaboju un iepušoju.